src/Pretty.hs

```diff
@@ -275,12 +275,8 @@ instance Pretty VarInfo SrcPos where
     viaShow (view srcPosCol pos)
 
 instance Pretty VarInfo a => Pretty VarInfo (Stx a) where
-  pp env (Stx _ loc v) =
-    text "#" <>
-    (align . group)
-      (text "[" <> pp env loc <> text "]" <> line' <> text "<" <>
-       align (pp env v) <>
-       text ">")
+  pp env (Stx _ _ v) =
+    text "'" <> pp env v
```

Comment:
This may be a controversial change. Variables used to be pretty-printed as `#[/.../file.kl:11.36-11.37]<foo>`, now I'm pretty-printing them as `'foo`. I imagine that some find the location information helpful when debugging, but I personally find it extremely unreadable. CI didn't like it either, because the golden files had my home folder baked in that `/.../file.kl` path. I could fix that by only including the basename of the file... but I find `'foo` a lot more readable. Let me know what you think!

Reply:
There was a good reason for sticking this information on them: it makes it possible to trace where they're from. And this change doesn't just strip that information from identifiers, it does it from _all_ syntax objects!

What about doing the following instead?
 * Show the paths relative to that of the current module being expanded (this should make them shorter and also fix the CI issue
 * Add a `Symbol` and/or `SyntaxContents` type, and then have a way to extract them from identifiers/arbitrary syntax objects, and use this in tests
 * Add an extra argument to `pp` that consists of pretty-printer options, and then have the display of source locations be an option that can be controlled by a command-line flag

Reply:
That current module isn't available in `pp`, so for now, I print the basename but not the folder. Is that good enough to merge this?